.circleci/config.yml

```diff
@@ -4,10 +4,13 @@ jobs:
     docker:
       - image: python:3.5
     working_directory: /home/ubuntu/useradm
+    dependencies:
+      pre:
+        - sudo apt-get update; sudo apt-get install
     steps:
       - checkout
-      - run: pip install -r requirements.txt
-      - run: pip install pylint
+      - run: pip3 install -r requirements.txt
```

Comment:
Trying to rule out things why requirements won't install. If I fix all these changes we have no PR. I don't plan to merge this. At this point it is just Debugging what is wrong